### PR TITLE
feat: includes proofs chains in the delegated authorization chain

### PR DESCRIPTION
**Pull request by Gozala**
This pr cleans up bunch of mess in the access/authorize code path and starts including proof chains in the delegated authorization so agents can actually utilize delegations.

### CODE REVIEW

File: packages/access-api/src/service/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export function service(ctx) {
       claim: (...args) => {
         // disable until hardened in test/staging
         if (ctx.config.ENV === 'production') {
-          throw new Error(`acccess/claim invocation handling is not enabled`)
+          throw new Error(`access/claim invocation handling is not enabled`)
```
Comment (gobengo) [inline packages/access-api/src/service/index.js:34]: thanks for fixing my typo :)

File: packages/access-api/test/helpers/ucanto-test-utils.js [hunk 2]
```diff
@@ -38,13 +38,14 @@ export function createTesterFromContext(createContext, options) {
  * using a service-issued voucher/redeem invocation
  *
  * @param {Iterable<Resolvable<Ucanto.Principal>>} spaces
- * @param {Ucanto.Signer<Ucanto.DID>} issuer
- * @param {Ucanto.ConnectionView<Record<string, any>>} conn
+ * @param {object} options
+ * @param {Ucanto.Signer<Ucanto.DID>} options.service
```
Comment (gobengo) [inline packages/access-api/test/helpers/ucanto-test-utils.js:42]: In the future I expect to have `spaceRegistrationInvocation` create a `provider/add` issued by an agent (not the 'service'), which is why I called it the more generic `issuer`. I can always change it back if/when we use non-service-issuers for this.

File: packages/capabilities/src/access.js [hunk 3]
```diff
@@ -74,33 +74,62 @@ export const authorize = capability({
...
+      }
     }],
     exp: null
     sig: "..."
   }
  * ```
  */
 export const session = capability({
-  can: './update',
+  can: 'ucan/attest',
```
Comment (gobengo) [inline packages/capabilities/src/access.js:127]: would be ideal to link to relevant spec for this, e.g. so someone could figure out how to build a valid/useful `Link` for `.nb.proof`

**Review by gobengo — APPROVED** [review]

File: packages/capabilities/src/access.js (hunk 3 above)
Comment (Gozala) [reply packages/capabilities/src/access.js:127]: The link on line 105 add this, or do you mean something else ?

**Pull request merged**